networking/src/main/java/ar/com/wolox/wolmo/networking/offline/Repository.java

```diff
@@ -114,7 +114,7 @@ public Repository(@NonNull C cache) {
      * @param queryStrategy that determines how to react to local/network actions
      */
     public Query<T> query(@AccessPolicy final int policy, @NonNull final Call<T> call,
-                          @NonNull final QueryStrategy<T, C> queryStrategy) {
+            @NonNull final QueryStrategy<T, C> queryStrategy) {
```

Comment:
Why did you remove this spacing? It existed to align the parameters.

Reply:
Lol, auto-formatting